lbrynet/daemon/DaemonCLI.py

```diff
@@ -38,8 +38,17 @@ def set_flag_vals(flag_names, parsed_args):
 
 
 def main():
-    if len(sys.argv[1:]):
-        method, args = sys.argv[1], sys.argv[2:]
+    argv = sys.argv[1:]
+
+    # check if a config file has been specified. If so, shift
+    # all the arguments so that the parsing can continue without
+    # noticing
+    if sys.argv[1] == "--conf":
+        conf.conf_file = sys.argv[2]
```

Comment:
this should check len(sys.argv) > 2 first in order to return a more meaningful error message if the conf file wasn't provided than an `IndexError`

Reply:
makes sense. Will fix!